cpp/tests/interpolate/cubic_spline_test.cpp

```diff
@@ -34,6 +34,14 @@
 struct CubicSplineTest : public cudf::test::BaseFixture {
 };
 
+// TEST_F(CubicSplineTest, test_parallel_search
```

Comment:
Delete this line? And explain what the numbered comments are.

Reply:
A comment for myself to remind me which tests to write, done now.